ios/FluentUI/SegmentedControl/SegmentedControl.swift

```diff
@@ -395,6 +419,7 @@ open class SegmentedControl: UIControl {
 
         if style != .tabs {
             layoutPillContainerView()
```

Comment:
talked offline bu we are going to set the constraint of the pill container and maskedlabelslayerview multiple time rather than once.